src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -42,21 +42,17 @@ public ErrorLogger(Stream stream, string toolName, string toolFileVersion, Versi
             _writer.WriteArrayStart("runs");
             _writer.WriteObjectStart(); // run
 
-            WriteToolInfo(toolName, toolFileVersion, toolAssemblyVersion);
+            _writer.WriteObjectStart("tool");
+            _writer.Write("name", toolName);
+            _writer.Write("version", toolAssemblyVersion.ToString());
+            _writer.Write("fileVersion", toolFileVersion);
+            _writer.Write("semanticVersion", toolAssemblyVersion.ToString(fieldCount: 3));
+            _writer.Write("language", culture.Name);
```

Comment:
"Usually"? ;-) Oh well, it's not like there's anything better you can do here...

---
In reply to: [63798378](https://github.com/dotnet/roslyn/pull/11390#discussion_r63798378) [](ancestors = 63798378)